software/last/lastdb/functions.nf

```diff
@@ -0,0 +1,60 @@
+/*
+ * -----------------------------------------------------
```

Comment:
This file needs to be replace with [this one](https://raw.githubusercontent.com/nf-core/modules/master/software/lib/functions.nf). It was updated recently.